SETUP.md

```diff
@@ -0,0 +1,32 @@
+# Installasjonsguide for workshop
+Guide for installasjon av editor, pakkebehandler og git.
```

Comment:
Det norske ordet for «editor» er kodeverktøy: https://www.statped.no/laringsressurs/syn/kodeverktoy-og-tilgjengelighet/?depth=0